### PR TITLE
Added option to import existing ssh public key

### PR DESCRIPTION
**Pull request by SweetOps**
## What

* Import existing ssh public key to AWS

## Why

* We need a way to import existing ssh key pair to AWS

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -0,0 +1,16 @@
+data "template_file" "default" {
+  template = "${file("${var.ssh_public_key_path}")}"
+}
+
+
+module "label" {
+  source    = "git::https://github.com/cloudposse/tf_label.git?ref=tags/0.2.0"
```
Comment (comeanother) [inline main.tf]: Add support of all `tf_label` `0.2.0` capabilities.

Comment (comeanother) [reply main.tf]: Apply `terraform plan`.

Comment (SweetOps) [reply main.tf]: has been fixed

**Comment by comeanother** [comment]
Add description to `ssh_public_key_path`
```
description = "Path to Read/Write SSH Public Key File (directory)"
```

**Comment by comeanother** [comment]
Update `tf_key_pair` according to this convention:
```
We would pass only a directory `ssh_public_key_path` with ssh key.
`tf_key-pair` will use `ssh_public_key_name` according to `label.id`
```

**Comment by SweetOps** [comment]
@comeanother , added by your request

File: main.tf [hunk 2]
```diff
@@ -0,0 +1,18 @@
+data "template_file" "default" {
+  template = "${file("${var.ssh_public_key_path}")}"
```
Comment (comeanother) [inline main.tf]: Update file path according to `${var.ssh_public_key_path}/${module.label.id}`

File: main.tf [hunk 3]
```diff
@@ -0,0 +1,18 @@
+data "template_file" "default" {
+  template = "${file("${var.ssh_public_key_path}/${module.label.id}")}"
```
Comment (comeanother) [inline main.tf]: add `.pub` at the end.

File: output.tf [hunk 4]
```diff
@@ -0,0 +1,3 @@
+output "ssh_key_name" {
```
Comment (osterman) [inline output.tf]: Drop `ssh_` prefix as it's implied by the module.

Comment (osterman) [reply output.tf]: Generally, try to mimic original variable name when it's not ambiguous.

File: variables.tf [hunk 5]
```diff
@@ -0,0 +1,15 @@
+variable "namespace" {}
+
+variable "stage" {}
+
+variable "name" {}
+
+variable "delimiter" {}
+
+variable "attributes" {}
```
Comment (comeanother) [inline variables.tf]: Use variable definitions of [tf_label](https://github.com/cloudposse/tf_label/blob/master/variables.tf)

**Pull request merged**